docs/plugins.md

```diff
@@ -11,7 +11,9 @@ Plugins are ways of adding new languages to Prettier. Prettier's own implementat
 
 ## Using Plugins
 
-Plugins are automatically loaded if you have them installed in the same `node_modules` directory where `prettier` is located. Plugin package names must start with `@prettier/plugin-` or `prettier-plugin-` to be registered.
+Plugins are automatically loaded if you have them installed in the same `node_modules` directory where `prettier` is located. Plugin package names must start with `@prettier/plugin-` or `prettier-plugin-` or `<scope>@prettier-plugin-` to be registered.
```

Comment:
This looks like it might be a typo

```suggestion
Plugins are automatically loaded if you have them installed in the same `node_modules` directory where `prettier` is located. Plugin package names must start with `@prettier/plugin-` or `prettier-plugin-` or `@<scope>/prettier-plugin-` to be registered.
```

Reply:
Oops, thanks for the review :)